sw/device/silicon_creator/lib/manifest.h

```diff
@@ -121,7 +122,13 @@ enum {
  */
 typedef struct manifest {
   /**
-   * Image signature.
+   * SPX signature of the image.
+   *
+   * TODO: Make this optional.
```

Comment:
Nit: there should be an issue number associated with this.

Reply:
Thanks for the reminder, created #17824.